hobo/script/orm_hobo.py

```diff
@@ -113,13 +102,24 @@ class ErrorLog(BASE):
     """
     __tablename__ = 'error_log'
 
+    class PipelineStageEnum(enum.Enum):
+        """
+        This class defines strings that could be inserted into error_log.pipeline_stage
+
+        Each string represents at what stage of the api script execution an error_log row was inserted
+            data_acquisition: obtaining readings from source
+            database_insertion: inserting new rows into readings table
+        """
+        data_acquisition = "data_acquisition"
+        database_insertion = "database_insertion"
+
     # the sqlalchemy orm requires a primary key in each table
     log_id = Column(Integer, primary_key=True)
     purpose_id = Column(Integer)
     datetime = Column(TIMESTAMP)
     status = Column(Boolean)
     error_type = Column(String)
-    pipeline_stage = Column(String)
+    pipeline_stage = Column(Enum(PipelineStageEnum))
```

Comment:
Sql Alchemy Enum.

Reply:
I could probably do something like this instead of using the Python enum classes.
```python3
pipeline_stage = Column(Enum('data_acquisition','database_insertion', name='pipeline_stage_enum'))
```

Reply:
We agreed they are different enums so this is needed.